virtual_drive/core.py

```diff
@@ -1400,6 +1404,9 @@ def _update_registers(self) -> None:
                 self.__reg_address_to_id[subnode][reg.address] = reg_id
                 self.__dictionary.registers(subnode)[reg_id].storage_valid = True
 
+        if not self.is_monitoring_available:
+            return
```

Comment:
I think we should create an `update_monitoring_disturbance_registers` method and call it when monitoring is available.

Reply:
Done [d1fc2fd](https://github.com/ingeniamc/ingenialink-python/pull/412/commits/d1fc2fd483295009d4486a4d8c8aa52aefb90ad5)